test/WebSites/ActionConstraintsWebSite/Startup.cs

```diff
@@ -11,11 +11,9 @@ public class Startup
     {
         public void Configure(IApplicationBuilder app)
         {
-            var configuration = app.GetTestConfiguration();
```

Comment:
no.  do not remove this line.  the method configures a required middleware for the functional tests.  we can rename it and make it `void` at some point.  but it must remain.